test/parallel/test-common.js

```diff
@@ -5,3 +5,12 @@ const assert = require('assert');
 common.globalCheck = false;
 global.gc = 42;  // Not a valid global unless --expose_gc is set.
 assert.deepStrictEqual(common.leakedGlobals(), ['gc']);
+
+
+assert.throws(function() {
+  common.mustCall(function() {}, 'foo')();
+}, /invalid expected value: foo/i);
```

Comment:
Can you do `/^TypeError: Invalid expected value: foo$/`. Also, drop the insensitive check.